### ISSUE TITLE
Add Algorithms and Data Structures in C Language. 

### ISSUE THREAD

**Issue by Anubhav-1020** [SanthanCH/DSA#1]
Hey @SanthanCH  If u allow .. I can add few Data Structures and Algorithms in C Language to this repo. I am new to open source this would be really helpful : )

**Comment by rohith1125** [comment]
@SanthanCH I wanna make some quality PR's too

**Comment by amrshindhe** [comment]
Hi can we create 10 problems each day

**Comment by amrshindhe** [comment]
Hi can we create 10 problems each day

**Comment by himik-nainwal** [comment]
Hi I am having quality codes for algorithms , I can contribute to it.
Thank you..

**Comment by SanthanCH** [comment]
> Hi can we create 10 problems each day

yeah we can do , I did some leetcode questions and I kept all codes in my competetive programming repository
have a look
[competetive programming](https://github.com/SanthanCH/Competitive-Programming)
